### ISSUE TITLE
Doesn't patch over a server side rendered content

### ISSUE THREAD

**Issue by graforlock** [snabbdom/snabbdom#335]
Having content like this in the body, rendered on the server (for example):

```html
<div>
  <h1>Hello world</h1>
  <button>Increment</button>
</div>
```

`patch` over div adds another duplicate content instead of correctly, deterministically, diffing over a container, so the output is in the end:

```html
<div>
  <h1>Hello world</h1>
  <button>Increment</button>
  <h1>Hello world</h1>
  <button>Increment</button>
</div>
```

Repo to reproduce:  https://github.com/graforlock/snabbdom-ssr-issue

I presume this is not a correct behaviour since it doesn't align with any isomorphic/SSR renderers and how snabbdom `toHTML` works

**Comment by caridy** [comment]
you need to look at `toVNode`, which was introduced recently. In your code, you should do:

```js
patch(toVNode(document.querySelector('div')), newNode)
```

__note: don't forget to import toVNode.__

I realized that we don't have `toVNode` documented in the README.md. @graforlock do you want to try that, and send us a PR for the README?

**Comment by mightyiam** [comment]
Closed in b8fcaf99a94c28c08b120b4be2748a0afddb86e3.
